man/import.q.sorts.Rd

```diff
@@ -17,69 +18,75 @@
   \item{q.sorts.dir}{
 		A relative path to a directory structure where:
 		\itemize{
-			\item folders are \code{conditions} (such as before or after), if there are more than one conditions.
-			\item filenames of \code{*.csv}s are participant names (might be given pseudneums)
-			\item csvs contain raw qsorts, beginning with all lines of items (not the values or), such as \code{,,item-12,item-64,,}
-			\item cells contain \emph{not} actual meaningful short handles, but some string to identify them as per \code{manual.lookup} option.
+			\item folders are \code{conditions} (such as \emph{before} or \emph{after}).
+			If there are no \code{conditions} (defaults to \code{NULL}), there should be no folders.
+			\item filenames of .CSVs are participant names (might be given pseudonyms).
```

Comment:
done in https://github.com/maxheld83/qmethod/commit/dec047f6fc92338f5eb5a4f4221cf046d323e6e6